lib/cocoapods/validator.rb

```diff
@@ -142,7 +142,12 @@ def failure_reason
         reasons << 'all results apply only to public specs, but you can use ' \
                    '`--private` to ignore them if linting the specification for a private pod'
       end
-      reasons.to_sentence
+      if dot_swift_version.nil?
+        reasons.to_sentence + ".\n[!] If you are trying to validate a Swift 3.0 Pod, " \
+          'you need to have a `.swift-version` file. e.g `echo "3.0" > .swift-version`'
+      else
+        reasons.to_sentence
+      end
     end
 
```

Comment:
Word wise, let's try aim for future proofing, e.g. explain why it is, then give the 3.0 example:

```

[!] The validator for Swift projects uses Swift 2.3 by default, if you are using a different version of swift
you can use a `.swift-version` file to set the version for your Pod. For example to use Swift 3.0, run:

`echo "3.0" > .swift-version` to create a version file in your repo.
```